<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Copmetitionswindow</class>
 <widget class="QDialog" name="Copmetitionswindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="labelCompetitions">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Competitions</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelBestRanked">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Best ranked competitions:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonAddCompetitions">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>40</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Competion</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonShowCompetitions">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>80</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Show Competions</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Name">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Date">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Date:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_NrOfEvents">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Number of events:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Placement">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Placement:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Description">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Description:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSave">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>230</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonGoBack">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Go back</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditName">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditDate">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPlace">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>180</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditNrOfEvents">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>150</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditDescription">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>220</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
